web/app.js

```diff
@@ -1873,8 +1885,28 @@ function handleMouseWheel(evt) {
   }
 }
 
-window.addEventListener('DOMMouseScroll', handleMouseWheel);
-window.addEventListener('mousewheel', handleMouseWheel);
+//#if !(CHROME || MOZCENTRAL)
+if (window.onwheel === undefined) {
+  // window.onwheel is not set, most likely wheel is not supported.
+  window.addEventListener('DOMMouseScroll', function (evt) {
```

Comment:
> What is the reason for keeping it around in this patch?

I'm assuming that it's for backwards compatibility.
However, I wonder if that's actually necessary here, given that the `wheel` event has been supported since Firefox 17 (https://developer.mozilla.org/en-US/docs/Web/Events/wheel#Browser_compatibility).